build_opencv.sh

```diff
@@ -1,7 +1,7 @@
-curl -L 'http://sourceforge.net/projects/opencvlibrary/files/opencv-unix/2.4.7/opencv-2.4.7.tar.gz/' | tar xvzf -
-mkdir -p opencv-2.4.7/release
-cd opencv-2.4.7/release
+curl -L 'https://github.com/Itseez/opencv/archive/2.4.11.zip' | unzip -p -
```

Comment:
http://unix.stackexchange.com/questions/2690/how-to-redirect-output-of-wget-as-input-to-unzip

according to this page, unzip from stdin is not yet supported